devel/make.xml

```diff
@@ -37,7 +40,20 @@ the Debian Free Software Guidelines, and has been removed from this package.</de
         <manifest-digest sha1new="90f22a15896ea14db005aa13b922b187f9fa8067" sha256="63bcce7ce44f11ec38b3f6ddc5f770bbffc5764febe1e8fb65b9f8b29b1d8336" sha256new="MO6M47HEJ4I6YOFT63O4L53QXP74K5SP5PQ6R63FXH4LFGY5QM3A"/>
         <archive href="http://downloads.sourceforge.net/gnuwin32/make-3.81-bin.zip" size="495645" type="application/zip"/>
       </implementation>
+
     </group>
+    <implementation rch="Windows-i486" license="GPL v2 (GNU General Public License)" id="withdeps" langs="be da de es fi fr ga gl he hr id ja ko nl pl pt_BR ru rw sv tr uk vi zh_CN" released="2006-11-25" version="3.81-3">
```

Comment:
I'd suggest removing the `langs` attribute as long as there are no other implementations with a different set of languages for 0install to potentially choose.

Reply:
done